### PR TITLE
Log exception rather than just passing

### PR DESCRIPTION
**Pull request by prigoyal**
show the error message explicitly, one user had wrong TC but we were not displaying the error message.

### CODE REVIEW

File: tensor_comprehensions/tc_unit.py [hunk 1]
```diff
@@ -204,7 +204,8 @@ def tune_and_store(self, tc_name, inputs, mapping_options, cache_file=""):
         try:
             best_options = self.autotuner.tune(cache_file, tc_name, inputs, options, [options])
             return best_options
-        except RuntimeError:
+        except Exception as e:
+            logger.error('Got exception: {}'.format(e))
```
Comment (wsmoses) [inline tensor_comprehensions/tc_unit.py]: is got the best word here? Maybe Raised?

Comment (prigoyal) [reply tensor_comprehensions/tc_unit.py]: yeah, will correct :) raised sounds better

**Comment by ftynse** [comment]
At some point, we discussed CHECK macros throwing exceptions instead of killing everything... This would make them visible in python too.

**Comment by prigoyal** [comment]
yep, all the exceptions thrown in C++ will be visible here which is I guess fine and users can provide info to us for debugging :)

**Pull request merged**